test/suites/network/offline.robot

```diff
@@ -7,6 +7,7 @@ Library             ../../resources/qemu-guest-agent.py
 Library             ../../resources/DataFormats.py
 Resource            ../../resources/libvirt.resource
 Resource            ../../resources/microshift-config.resource
+Resource            ../../resources/microshift-process.resource
```

Comment:
```suggestion
Resource            ../../resources/ostree-health.resource
```
And use `Wait Until Greenboot Health Check Exited` function.

Reply:
thanks, addressed in the last commit